manifest.php

```diff
@@ -29,7 +29,7 @@
     'label' => 'Synchronization Client',
     'description' => 'Synchronization logic specific only for the client server',
     'license' => 'GPL-2.0',
-    'version' => '0.2.1',
+    'version' => '0.2.2',
```

Comment:
@Yury-Razhkou version updates only once in the PR, I guess for this PR it has been bumped already

Reply:
0.2.0 is the correct version for this PR

Reply:
it is for triggering update without reinstalling from scratch.